PrivatePing/settings/base.py

```diff
@@ -95,4 +95,6 @@
     os.path.join(BASE_DIR, '../assets/static')
 ]
 STATIC_ROOT = os.path.join(BASE_DIR, '../assets/')
-STATIC_URL = '/static/'
\ No newline at end of file
+STATIC_URL = '/static/'
+
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```

Comment:
I find only this one important.
no need to change anything in assets.
if you think, changes in assets are necessary, kindly explain.

Reply:
Yes yes, I overlooked the importance of WhiteNoise in my last commit. I have reverted back the assets but only restructured the directories to ensure proper static file handling, where I separated the staticfiles from static